invenio_rdm_records/theme/templates/invenio_rdm_records/details/side_bar.html

```diff
@@ -0,0 +1,124 @@
+{#
+  Copyright (C) 2020 CERN.
+  Copyright (C) 2020 Northwestern University.
+
+  Invenio RDM Records is free software; you can redistribute it and/or modify
+  it under the terms of the MIT License; see LICENSE file for more details.
+#}
+<dt class="record-info-title">{{ _('Metrics')}}:</dt>
+<hr class="thin-line"></hr>
+<dd>
+<!-- Stats -->
+<div class="row stats-box">
+  <div id="accordion">
+      <!-- Banner -->
+      <div class="row">
+          <div class="col-sm-4">
+              <span class="stats-data">1,175</span>
+          </div>
+          <div class="col-sm-4">
+              <span class="stats-data">52</span>
+          </div>
+          <div class="col-sm-4">
+              <span class="stats-data">8</span>
+          </div>
+      </div>
+      <div class="row">
+          <div class="col-sm-4">
+              <i class="fa fa-eye " data-toggle="tooltip" tooltip="" data-placement="top" title="" data-original-title="Total views."></i> views
+          </div>
+          <div class="col-sm-4">
+              <i class="fa fa-download " data-toggle="tooltip" tooltip="" data-placement="top" title="" data-original-title="Total downloads."></i> downloads
+          </div>
+          <div class="col-sm-4">
+              <i class="fa fa-pencil " data-toggle="tooltip" tooltip="" data-placement="top" title="" data-original-title="Total citation."></i> citations
+          </div>
+      </div>
+
+    <!-- Collapsed details -->
+    <div id="toggle-stats" class="row">
+      <div class="col-sm-12">
+        <a style="cursor: pointer;" class="panel-toggle" data-toggle="collapse" data-target="#collapse-stats" aria-expanded="true" aria-controls="collapse-stats">
+          See more details...
```

Comment:
should say "See less details" when unrolled. This can be shelved for when we integrate invenio-stats.

Reply:
Done :) tho to be re-checked when integrating invenio-stats :)